domain/src/test/java/org/oppia/android/domain/oppialogger/analytics/AnalyticsControllerTest.kt

```diff
@@ -73,11 +93,32 @@ private const val TEST_QUESTION_ID = "test_questionId"
 private const val TEST_SKILL_ID = "test_skillId"
 private const val TEST_SKILL_LIST_ID = "test_skillListId"
 private const val TEST_SUB_TOPIC_ID = 1
+private const val TEST_LEARNER_ID = "test_learnerId"
+private const val TEST_DEVICE_ID = "test_deviceId"
+private const val TEST_SESSION_ID = "test_sessionId"
+private const val TEST_EXPLORATION_VERSION = "test_exploration_version"
+private const val TEST_STATE_NAME = "test_state_name"
+private const val TEST_HINT_INDEX = 0
+private const val TEST_IS_ANSWER_CORRECT = true
+private const val TEST_CONTENT_ID = "test_contentId"
 
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(application = AnalyticsControllerTest.TestApplication::class)
 class AnalyticsControllerTest {
+  private companion object {
```

Comment:
Similarly, there are changes here that need to be cleaned up and/or moved to #4248.